MovieApp/MovieApp/CustomTableViewCell.h

```diff
@@ -0,0 +1,19 @@
+//
+//  CustomTableViewCell.h
+//  MovieApp
+//
+//  Created by user on 26/09/16.
+//  Copyright © 2016 internshipABH. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface CustomTableViewCell : UITableViewCell
+
+@property (nonatomic, readonly) CGFloat height;
+
+- (CGFloat)getHeight;
++(CGFloat)cellHeight;
++(NSString*)cellIdentifier;
```

Comment:
Remove `getHeight` method, since it is unused. Implement `cellHeight` and `cellIdentifier` methods by setting default values (that can be overridden).